observability/logging/logging_release_notes/cluster-logging-deploying.adoc

```diff
@@ -0,0 +1,48 @@
+:_mod-docs-content-type: ASSEMBLY
+:context: cluster-logging-deploying
+[id="cluster-logging-deploying"]
+= Installing Logging
+include::_attributes/common-attributes.adoc[]
+include::_attributes/attributes-openshift-dedicated.adoc[]
+
+toc::[]
+
+{Product-title} Operators use custom resources (CR) to manage applications and their components. High-level configuration and settings are provided by the user within a CR. The Operator translates high-level directives into low-level actions, based on best practices embedded within the Operator’s logic. A custom resource definition (CRD) defines a CR and lists all the configurations available to users of the Operator. Installing an Operator creates the CRDs, which are then used to generate CRs.
+
+[IMPORTANT]
+====
+You must install the {clo} *after* the log store Operator.
```

Comment:
We typically would not bold words to call attention to them. I think the bold highlighting could be dropped in favour of the Important note being enough.